daemons/fenced/pacemaker-fenced.c

```diff
@@ -1489,8 +1489,8 @@ main(int argc, char **argv)
         printf
             (_("    <longdesc lang=\"en\">Some devices do not support the standard 'port' parameter or may provide additional ones.\n"
              "Use this to specify an alternate, device-specific, parameter that should indicate the machine to be fenced.\n"
-             "A value of '" PCMK__VALUE_NONE "' can be used to tell the "
-             "cluster not to supply any additional parameters.\n"
```

Comment:
It would also be fine to use '%s' here and substitute PCMK__VALUE_NONE in as long as you do that outside of the _() call.

Reply:
It's a good question how defined constants should be handled. They should always be magic strings so I suppose they should always be untranslated. The %s trick makes sense though it means the zh translation for this will need to be updated ...